tests/acceptance/api_test.go

```diff
@@ -1766,9 +1766,12 @@ func TestApi(t *testing.T) {
 			// Verify error response json
 			// Get nested json in response json
 			err := data["error"].(map[string]interface{})
-			errStr := fmt.Sprintf("Unable to create lease: User principal %s has already spent 0.000000 of their principal budget", principalID)
 			require.Equal(t, "RequestValidationError", err["code"].(string))
-			require.Equal(t, errStr, err["message"].(string))
+			require.Equal(t,
```

Comment:
This test was expecting buggy behavior (that budget=0). That buggy behavior was fixed in the PR, so test is updated.
Also, cleaned up the error response message